resources/Hooks/useCountdown.js

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from 'react'
+
+const useCountdown = (targetDate) => {
+  const countDownDate = new Date(targetDate).getTime()
+
+  const [countDown, setCountDown] = useState(countDownDate - new Date().getTime())
+
+  useEffect(() => {
+    const interval = setInterval(() => {
```

Comment:
will this create a new interval every time countdown date changes or does the previous one get cleared?